### PR TITLE
CI/Bors fixes

### PR DESCRIPTION
**Pull request by bonomat**

### CODE REVIEW

File: .github/workflows/system_tests.yml [hunk 1]
```diff
@@ -25,7 +25,7 @@ jobs:
         run: cargo install flutter_rust_bridge_codegen --version 1.62.1
       - name: Start containers
         run: |
-          docker-compose up bitcoin electrs -d
+          docker-compose up
```
Comment (holzeis) [inline .github/workflows/system_tests.yml:28]: The services have to come after the options - `docker compose up [OPTIONS] [SERVICE...]`

Also I guess we need chopsticks for the system tests. So `docker-compose up -d chopsticks` should be fine. Note, we can skip the `bitcoin` and `electrs` service as these are dependencies of `chopsticks` and will get started anways.

**Review by 1010Tom — APPROVED** [review]
Imho this is fine if it works.

**Comment by 1010Tom** [comment]
Going ahead and merge this is it is not controversial.

bors r+

**Comment by 1010Tom** [comment]
Going ahead and merge this is it is not controversial.

@bonomat : I hope this is fine.

bors r+

**Pull request merged**